python/lsst/ts/electrometer/controller.py

```diff
@@ -316,7 +338,12 @@ async def start_scan_dt(self, scan_duration):
         await self.send_command(self.commands.set_timer(self.mode))
         await self.send_command(self.commands.enable_sync(False))
         await self.send_command(f"{self.commands.clear_buffer()}")
-        await self.send_command(f"{self.commands.format_trac()}")
+        format_trac_args = {}
```

Comment:
This is duplicated from lines 314 to 319. Please extract to a function to avoid code duplication.